### DISCUSSION TITLE
Analog Input

### DISCUSSION THREAD

**Discussion by FannyCazettes** [harp-tech/protocol#80] [Q&A] [answered]
Hi, 
We have a rotary encoder with an analog output. Can the Behavior board read the analog signals?
Otherwise do you have another board that would convert analog to PWM or any other suggestion to make this work?
Thanks!

**Comment by bruno-f-cruz** [comment, marked as answer]
Yes, the Behavior board can read analog inputs. This is available in one in single input port for hw version <1.2 and two analog input ports for hw  version >2.0.

The inputs expect a signal from 0-5V, have a 12bit ADC, and sample at 1kHz.

**Comment by FannyCazettes** [comment]
Thanks Bruno!
